rtc-video-quality/setup_vmaf.sh

```diff
@@ -23,7 +23,6 @@ fi
 # Check out the pinned vmaf version.
 pushd vmaf
 git fetch
-git checkout --detach 45c57f7b67cebc301d85715669b9126063903ac2
```

Comment:
Don't delete this line. Like our other libraries, we should point to a specific version so that any updates to vmaf library don't change our results between runs. Let's update this the most current version.

We can manually update the vmaf version here when we find useful updates.